[2024-08-12T18:04:47.757+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: monthly_monitor.create_db scheduled__2024-04-01T00:00:00+00:00 [queued]>
[2024-08-12T18:04:47.780+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: monthly_monitor.create_db scheduled__2024-04-01T00:00:00+00:00 [queued]>
[2024-08-12T18:04:47.782+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2024-08-12T18:04:47.896+0000] {taskinstance.py:1380} INFO - Executing <Task(_PythonDecoratedOperator): create_db> on 2024-04-01 00:00:00+00:00
[2024-08-12T18:04:47.987+0000] {standard_task_runner.py:57} INFO - Started process 7837 to run task
[2024-08-12T18:04:47.998+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'monthly_monitor', 'create_db', 'scheduled__2024-04-01T00:00:00+00:00', '--job-id', '204', '--raw', '--subdir', 'DAGS_FOLDER/monitor.py', '--cfg-path', '/tmp/tmpermmyoe8']
[2024-08-12T18:04:48.044+0000] {standard_task_runner.py:85} INFO - Job 204: Subtask create_db
[2024-08-12T18:04:48.439+0000] {task_command.py:415} INFO - Running <TaskInstance: monthly_monitor.create_db scheduled__2024-04-01T00:00:00+00:00 [running]> on host a9166c5dc5ae
[2024-08-12T18:04:49.119+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='monthly_monitor' AIRFLOW_CTX_TASK_ID='create_db' AIRFLOW_CTX_EXECUTION_DATE='2024-04-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-01T00:00:00+00:00'
[2024-08-12T18:04:49.168+0000] {taskinstance.py:1935} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/base.py", line 221, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/monitor.py", line 31, in create_db
    conn = psycopg2.connect(**conn_params)
  File "/home/airflow/.local/lib/python3.8/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: connection to server at "localhost" (127.0.0.1), port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
connection to server at "localhost" (::1), port 5432 failed: Cannot assign requested address
	Is the server running on that host and accepting TCP/IP connections?

[2024-08-12T18:04:49.269+0000] {taskinstance.py:1398} INFO - Marking task as FAILED. dag_id=monthly_monitor, task_id=create_db, execution_date=20240401T000000, start_date=20240812T180447, end_date=20240812T180449
[2024-08-12T18:04:49.360+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 204 for task create_db (connection to server at "localhost" (127.0.0.1), port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
connection to server at "localhost" (::1), port 5432 failed: Cannot assign requested address
	Is the server running on that host and accepting TCP/IP connections?
; 7837)
[2024-08-12T18:04:49.405+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-08-12T18:04:49.672+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-12T18:13:33.570+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: monthly_monitor.create_db scheduled__2024-04-01T00:00:00+00:00 [queued]>
[2024-08-12T18:13:33.666+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: monthly_monitor.create_db scheduled__2024-04-01T00:00:00+00:00 [queued]>
[2024-08-12T18:13:33.697+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2024-08-12T18:13:33.848+0000] {taskinstance.py:1380} INFO - Executing <Task(_PythonDecoratedOperator): create_db> on 2024-04-01 00:00:00+00:00
[2024-08-12T18:13:33.986+0000] {standard_task_runner.py:57} INFO - Started process 9101 to run task
[2024-08-12T18:13:33.995+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'monthly_monitor', 'create_db', 'scheduled__2024-04-01T00:00:00+00:00', '--job-id', '255', '--raw', '--subdir', 'DAGS_FOLDER/monitor.py', '--cfg-path', '/tmp/tmp81elwhvs']
[2024-08-12T18:13:34.021+0000] {standard_task_runner.py:85} INFO - Job 255: Subtask create_db
[2024-08-12T18:13:34.338+0000] {task_command.py:415} INFO - Running <TaskInstance: monthly_monitor.create_db scheduled__2024-04-01T00:00:00+00:00 [running]> on host a9166c5dc5ae
[2024-08-12T18:13:34.871+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='monthly_monitor' AIRFLOW_CTX_TASK_ID='create_db' AIRFLOW_CTX_EXECUTION_DATE='2024-04-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-01T00:00:00+00:00'
[2024-08-12T18:13:34.936+0000] {logging_mixin.py:151} INFO - churn database already created
[2024-08-12T18:13:34.942+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-08-12T18:13:35.010+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=monthly_monitor, task_id=create_db, execution_date=20240401T000000, start_date=20240812T181333, end_date=20240812T181335
[2024-08-12T18:13:35.123+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-08-12T18:13:35.272+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-12T19:22:00.080+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: monthly_monitor.create_db scheduled__2024-04-01T00:00:00+00:00 [queued]>
[2024-08-12T19:22:00.173+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: monthly_monitor.create_db scheduled__2024-04-01T00:00:00+00:00 [queued]>
[2024-08-12T19:22:00.180+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2024-08-12T19:22:00.209+0000] {taskinstance.py:1380} INFO - Executing <Task(_PythonDecoratedOperator): create_db> on 2024-04-01 00:00:00+00:00
[2024-08-12T19:22:00.250+0000] {standard_task_runner.py:57} INFO - Started process 4254 to run task
[2024-08-12T19:22:00.271+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'monthly_monitor', 'create_db', 'scheduled__2024-04-01T00:00:00+00:00', '--job-id', '479', '--raw', '--subdir', 'DAGS_FOLDER/monitor.py', '--cfg-path', '/tmp/tmpuyn7eymu']
[2024-08-12T19:22:00.283+0000] {standard_task_runner.py:85} INFO - Job 479: Subtask create_db
[2024-08-12T19:22:00.540+0000] {task_command.py:415} INFO - Running <TaskInstance: monthly_monitor.create_db scheduled__2024-04-01T00:00:00+00:00 [running]> on host a9166c5dc5ae
[2024-08-12T19:22:00.927+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='monthly_monitor' AIRFLOW_CTX_TASK_ID='create_db' AIRFLOW_CTX_EXECUTION_DATE='2024-04-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-01T00:00:00+00:00'
[2024-08-12T19:22:00.977+0000] {logging_mixin.py:151} INFO - churn database already created
[2024-08-12T19:22:00.982+0000] {python.py:194} INFO - Done. Returned value was: {'host': 'postgres', 'port': '5432', 'user': '***', 'password': '***', 'dbname': 'churn'}
[2024-08-12T19:22:01.101+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=monthly_monitor, task_id=create_db, execution_date=20240401T000000, start_date=20240812T192200, end_date=20240812T192201
[2024-08-12T19:22:01.212+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-08-12T19:22:01.393+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-12T19:27:13.068+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: monthly_monitor.create_db scheduled__2024-04-01T00:00:00+00:00 [queued]>
[2024-08-12T19:27:13.081+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: monthly_monitor.create_db scheduled__2024-04-01T00:00:00+00:00 [queued]>
[2024-08-12T19:27:13.090+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2024-08-12T19:27:13.182+0000] {taskinstance.py:1380} INFO - Executing <Task(_PythonDecoratedOperator): create_db> on 2024-04-01 00:00:00+00:00
[2024-08-12T19:27:13.243+0000] {standard_task_runner.py:57} INFO - Started process 5247 to run task
[2024-08-12T19:27:13.262+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'monthly_monitor', 'create_db', 'scheduled__2024-04-01T00:00:00+00:00', '--job-id', '534', '--raw', '--subdir', 'DAGS_FOLDER/monitor.py', '--cfg-path', '/tmp/tmpkmqwt_v5']
[2024-08-12T19:27:13.283+0000] {standard_task_runner.py:85} INFO - Job 534: Subtask create_db
[2024-08-12T19:27:13.499+0000] {task_command.py:415} INFO - Running <TaskInstance: monthly_monitor.create_db scheduled__2024-04-01T00:00:00+00:00 [running]> on host a9166c5dc5ae
[2024-08-12T19:27:13.855+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='monthly_monitor' AIRFLOW_CTX_TASK_ID='create_db' AIRFLOW_CTX_EXECUTION_DATE='2024-04-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-01T00:00:00+00:00'
[2024-08-12T19:27:13.892+0000] {logging_mixin.py:151} INFO - churn database already created
[2024-08-12T19:27:13.907+0000] {python.py:194} INFO - Done. Returned value was: {'host': 'postgres', 'port': '5432', 'user': '***', 'password': '***', 'dbname': 'churn'}
[2024-08-12T19:27:14.032+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=monthly_monitor, task_id=create_db, execution_date=20240401T000000, start_date=20240812T192713, end_date=20240812T192714
[2024-08-12T19:27:14.155+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-08-12T19:27:14.595+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-12T19:28:37.153+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: monthly_monitor.create_db scheduled__2024-04-01T00:00:00+00:00 [queued]>
[2024-08-12T19:28:37.180+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: monthly_monitor.create_db scheduled__2024-04-01T00:00:00+00:00 [queued]>
[2024-08-12T19:28:37.184+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2024-08-12T19:28:37.229+0000] {taskinstance.py:1380} INFO - Executing <Task(_PythonDecoratedOperator): create_db> on 2024-04-01 00:00:00+00:00
[2024-08-12T19:28:37.276+0000] {standard_task_runner.py:57} INFO - Started process 5755 to run task
[2024-08-12T19:28:37.305+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'monthly_monitor', 'create_db', 'scheduled__2024-04-01T00:00:00+00:00', '--job-id', '572', '--raw', '--subdir', 'DAGS_FOLDER/monitor.py', '--cfg-path', '/tmp/tmps842udiu']
[2024-08-12T19:28:37.319+0000] {standard_task_runner.py:85} INFO - Job 572: Subtask create_db
[2024-08-12T19:28:37.437+0000] {task_command.py:415} INFO - Running <TaskInstance: monthly_monitor.create_db scheduled__2024-04-01T00:00:00+00:00 [running]> on host a9166c5dc5ae
[2024-08-12T19:28:37.634+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='monthly_monitor' AIRFLOW_CTX_TASK_ID='create_db' AIRFLOW_CTX_EXECUTION_DATE='2024-04-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-01T00:00:00+00:00'
[2024-08-12T19:28:37.648+0000] {logging_mixin.py:151} INFO - churn database already created
[2024-08-12T19:28:37.650+0000] {python.py:194} INFO - Done. Returned value was: {'host': 'postgres', 'port': '5432', 'user': '***', 'password': '***', 'dbname': 'churn'}
[2024-08-12T19:28:37.682+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=monthly_monitor, task_id=create_db, execution_date=20240401T000000, start_date=20240812T192837, end_date=20240812T192837
[2024-08-12T19:28:37.742+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-08-12T19:28:37.863+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-12T20:09:38.886+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: monthly_monitor.create_db scheduled__2024-04-01T00:00:00+00:00 [queued]>
[2024-08-12T20:09:38.978+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: monthly_monitor.create_db scheduled__2024-04-01T00:00:00+00:00 [queued]>
[2024-08-12T20:09:39.027+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2024-08-12T20:09:39.115+0000] {taskinstance.py:1380} INFO - Executing <Task(_PythonDecoratedOperator): create_db> on 2024-04-01 00:00:00+00:00
[2024-08-12T20:09:39.194+0000] {standard_task_runner.py:57} INFO - Started process 13021 to run task
[2024-08-12T20:09:39.207+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'monthly_monitor', 'create_db', 'scheduled__2024-04-01T00:00:00+00:00', '--job-id', '887', '--raw', '--subdir', 'DAGS_FOLDER/monitor.py', '--cfg-path', '/tmp/tmp3zmx_vjp']
[2024-08-12T20:09:39.213+0000] {standard_task_runner.py:85} INFO - Job 887: Subtask create_db
[2024-08-12T20:09:39.649+0000] {task_command.py:415} INFO - Running <TaskInstance: monthly_monitor.create_db scheduled__2024-04-01T00:00:00+00:00 [running]> on host a9166c5dc5ae
[2024-08-12T20:09:41.410+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='monthly_monitor' AIRFLOW_CTX_TASK_ID='create_db' AIRFLOW_CTX_EXECUTION_DATE='2024-04-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-01T00:00:00+00:00'
[2024-08-12T20:09:41.524+0000] {logging_mixin.py:151} INFO - churn database already created
[2024-08-12T20:09:41.527+0000] {python.py:194} INFO - Done. Returned value was: {'host': 'postgres', 'port': '5432', 'user': '***', 'password': '***', 'dbname': 'churn'}
[2024-08-12T20:09:41.677+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=monthly_monitor, task_id=create_db, execution_date=20240401T000000, start_date=20240812T200938, end_date=20240812T200941
[2024-08-12T20:09:41.821+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-08-12T20:09:42.318+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
